acceptance/tests/fixtures/bases/api-gateway-v2/meshservice.yaml

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+apiVersion: consul.hashicorp.com/v1alpha1
```

Comment:
I'm not sure we need the MeshService here because we're not talking to any services outside of the cluster